setup-scripts/setup-tpm2-tools.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+
 set -euo pipefail 
 
-sudo apt -y install tpm2-tools qrencode
+# Where libengine-tpm2-tss-openssl lives
+sudo sh -c 'echo "\ndeb http://http.us.debian.org/debian bullseye-backports main" >> /etc/apt/sources.list'
```

Comment:
We might want to be more careful about throwing in backports like this. I would recommend (for now) adding it but pinning its priority to really low, and then using `apt install -t bullseye-backports libengine-tpm2-tss-openssl` to make sure that *only* that package is retrieved from backports. 

The other potential complication here is that if we bump versions to Debian 12, the backport will break (tho it will not be necessary any more, since the stuff in bullseye-backports is stuff that's in 12 that is ported back to 11).

Reply:
Good call! Took the approach that you suggested for now: https://github.com/votingworks/vxsuite-complete-system/pull/302/commits/0dc4f14f01ebaed0b62874176c35c88202d6ab25